### ISSUE TITLE
STPImageLibrary.cardBrandImage doesn't work in SPM

### ISSUE THREAD

**Issue by iKorich** [stripe/stripe-ios#1857]
Wrong path for resources in
var image = UIImage(
        named: imageName, in: StripeBundleLocator.resourcesBundle, compatibleWith: nil)
image == nil

## Summary
Failed in
assert(image.size != .zero, "Failed to find an image named \(imageName)")

## Code to reproduce
STPImageLibrary.cardBrandImage(for: .visa)

## iOS version
14.6

## Installation method
SPM

## SDK version
21.8.0

## Other information
lib search resources in Stripe_StripeCore.bundle instead of Stripe_Stripe.bundle

**Comment by davidme-stripe** [comment]
Thank you for filing this, and sorry for the trouble! We'll have a fix out very soon.

**Comment by davidme-stripe** [comment]
This has been fixed in `21.8.1`.

**Comment by chrisjenx** [comment]
Still broken for me, getting:

```
assert(image.size != .zero, "Failed to find an image named \(imageName)")
```
